docs/monitoring.md

```diff
@@ -28,4 +29,15 @@ Currently these dashboards have been applied manually; e.g. with
 ```
 # apply dashboard
 kubectl apply -n laa-court-data-ui-staging -f .k8s/staging/http_status_codes_grafana_dashboard.yaml
-```
\ No newline at end of file
+```
+
+### Alerting
+
+PrometheusRules for AlertManager alert conditions are defined in prometheus-custom-rules.yaml for each namespace. These are integrated with Slack using a slack webhook contained in the lcdui-secrets#SLACK_ALERTS_WEBHOOK secret.
+
+Currently these rules have been applied manually: e.g. with
```

Comment:
lets discuss whether we need to or should have a more automated approach to applying the rules